docs/Contribute.ipynb

```diff
@@ -22,8 +22,8 @@
     "> Warning: Don't confuse git and GitHub. Git is a tool used for version control. GitHub is an online platform used as a remote repository for git projects.\n",
```

Comment:
<p>Instead of <code>git checkout dev_branch_name</code> let's be consistent with above and do <code>git checkout branch_name</code> instead</p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/416/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>